script-class.js

```diff
@@ -42,3 +43,18 @@ document.querySelector('.menu').querySelectorAll('li').forEach((link) => {
     displayPage(e.target.id);
   });
 });
+
+
+function updateClock() {
+  const dt = new Date();
+  const DivDate =  document.querySelector('.date-time');
+  DivDate.innerHTML = dt.toLocaleString();  
+}
+
+function initClock() {
+  updateClock();
+  window.setInterval(updateClock, 1);
+}
+
+window.onload = initClock;
```

Comment:
- [ ] Good job implementing your own function to add a clock to the page, unfortunately, part of the requirements of this project is that you use an external library to add function to your webpage, in this case, [Luxon.](https://moment.github.io/luxon/#/install) [You can also read more about working with Luxon here.](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time)